CONFIGURATION.md

```diff
@@ -0,0 +1,182 @@
+|-------------------------------|-----------------------------------------------------|-------------------------------------------|----------|
+| `mail.from`                          | Sender email address for wave notifications.  |       | false    |
+
+
+## Jackson configuration
+
+| Variable                      | Description                                         | Default Value / Environment Variable | Optional |
+|-------------------------------|-----------------------------------------------------|-------------------------------------------|----------|
+| `jackson.serialization.writeDatesAsTimestamps` | Whether to write dates as timestamps in jackson serialization. | `false`                                  | false    |
+
+
+## Micronaut specific Configuration
+
+| Variable                    | Description                                                                                                                                                                                                                                                                   | Default Value / Environment Variable | Optional |
+|-----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------|----------|
+| `micronaut.caches.cache-20sec.expire-after-access` | Cache with a 20-second expire-after-access policy.                                                                                                                                                                                                                            | `20s`                                     | false    |
+| `micronaut.caches.cache-1min.expire-after-access`  | Cache with a 1-minute expire-after-access policy.                                                                                                                                                                                                                             | `1m`                                      | false    |
+| `micronaut.server.thread-selection` | Thread selection strategy.                                                                                                                                                                                                                                                    | `AUTO`                                    | false    |
+| `micronaut.server.idle-timeout` | Idle timeout for the server. note that this should be greater thean or equal the build timeout (which is 15 minutes)                                                                                                                                                          | `910s`                     | false    |
```

Comment:
```suggestion
| `micronaut.server.idle-timeout` | Idle timeout for the server. Note that this should be greater than or equal to the build timeout (15 minutes by default)                                                                                                                                                          | `910s`                     | false    |
```

`wave.build.timeout` is set to `5m` by default, so where is that 15 minutes coming from?

Reply:
removed from configuration